Modules.md

```diff
@@ -0,0 +1,112 @@
+# Modules
+
+The distributable, loadable, and executable unit of code in WebAssembly
+is called a **module**. A module contains:
+* a set of [imports and exports](Modules.md#imports-and-exports);
+* a section defining the [initial state of linear memory](Modules.md#initial-state-of-linear-memory);
+* a section containing [code](Modules.md#code-section);
+* after the MVP, sections containing [debugging/symbol information](Tooling.md); and
```

Comment:
I was discussing this with @kg yesterday: debug info would be nice as a separate file too. Not to design a future feature here, but maybe "sections contained debugging/symbol information, or a reference to separate files containing them"?